apps/src/blockTooltips/DropletBlockTooltipManager.js

```diff
@@ -87,6 +87,13 @@ DropletBlockTooltipManager.prototype.installTooltipsForCurrentCategoryBlocks_ =
       const hoverDivRect = blockHoverDiv.getBoundingClientRect();
       const toolboxRight = $('.droplet-palette-scroller').width();
       const offsetX = Math.min(hoverDivRect.width, toolboxRight);
+      if (offsetX === 0) {
```

Comment:
Is it possible offsetX could ever be 0 at a time when we _would_ want to render the tooltip?

Reply:
I haven't been able to find a case like that so far. Curious if you have any suggestions?